### PR TITLE
Update owncloud tarball url

### LINKED ISSUE #22: Nightly is failing
The nightlies were failing with the following message:

```bash
latest: Pulling from owncloud/ubuntu
latest: Pulling from owncloud/ubuntu
Digest: sha256:ed1906de4ef37b9b82f3b967c5f735b1ae2965fecc442a189a133e0bd85d1d71
Status: Image is up to date for owncloud/ubuntu:latest
+ tar -xjf owncloud.tar.bz2 -C /var/www
bzip2: (stdin) is not a bzip2 file.
tar: Child returned status 2
tar: Error is not recoverable: exiting now
```

Same as https://github.com/owncloud/litmus-testing/issues/7

### PR DESCRIPTION
**Pull request by saw-jan**
Now, `download.owncloud.org` is redirected to `download.owncloud.com`. And the new location for the daily tarballs is in `https://download.owncloud.com/server/daily/`

Fixes #22

Changes:
- use `owncloudci/core` image to download the owncloud
- added `cancel-previous build`
- notification enabled (to `builds` channel)
- removed client version `2.10` and added version `2.11`

### CODE REVIEW

File: .drone.star [hunk 1]
```diff
@@ -146,25 +158,70 @@ def smashbox(ctx, client_version, server_version, test_suite):
...
+        },
+        "steps": [
+            {
+                "name": "notify-rocketchat",
+                "image": PLUGINS_SLACK,
+                "settings": {
+                    "webhook": {
+                        "from_secret": "private_rocketchat",
+                    },
+                    "channel": "builds",
```
Comment (saw-jan) [inline .drone.star:200]: Is it OK to send build status to `builds` channel?

File: .drone.star [hunk 2]
```diff
@@ -1,13 +1,22 @@
...
         {
             "value": "daily-master",
-            "tarball": "https://download.owncloud.org/community/owncloud-daily-master.tar.bz2",
+            "tarball": "https://download.owncloud.com/server/daily/owncloud-daily-master.tar.bz2",
             # 'sha256': '488fc136a8000b42c447e224e69f16f5c31a231eea3038614fdea5e0c2218457',
         },
     ]
 
-    client_versions = ["2.10", "3.0"]
+    client_versions = ["2.11", "3.0"]
```
Comment (saw-jan) [inline .drone.star]: url to `2.10` is not available in https://download.owncloud.com/desktop/ownCloud/daily/

**Comment by saw-jan** [comment]
Can the tests be fixed?
https://drone.owncloud.com/owncloud/smashbox-testing/2021

CC @TheOneRing

**Comment by TheOneRing** [comment]
👍

**Comment by TheOneRing** [comment]
I guess unzip isn't installed, at least that's what the error indicates.

**Comment by saw-jan** [comment]
> I guess unzip isn't installed, at least that's what the error indicates.

Installing unzip works for client 2.11 tests only.
Now, fixes needed for client 3.0

**Comment by saw-jan** [comment]
While running tests against `client 3.0`, command `/usr/bin/owncloudcmd --trust` crashes. Here are the logs:

<details>
<summary>Log</summary>

```bash
2022-09-28 03:44:53,326 - INFO - sharer - running '/usr/bin/owncloudcmd --trust /smashdir/test_reshareDir/sharer/ owncloud://test_reshareDir1:owncloud@owncloud-server:8080/remote.php/webdav/ >> /smashdir/test_reshareDir/sharer-ocsync.step03.cnt000.log 2>&1'
2022-09-28 03:44:53,344 - WARNING - sharer - Non-zero exit code 1 from command '/usr/bin/owncloudcmd --trust /smashdir/test_reshareDir/sharer/ owncloud://test_reshareDir1:owncloud@owncloud-server:8080/remote.php/webdav/ >> /smashdir/test_reshareDir/sharer-ocsync.step03.cnt000.log 2>&1'
...
Segmentation fault (core dumped)
```
</details>

<details>
<summary>Coredump</summary>

```bash
Excess command line arguments ignored. (full)
[New LWP 1094]
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".
Core was generated by `/usr/bin/owncloudcmd --trust /smashdir/test_reshareDir/sharer/ owncloud://test_'.
Program terminated with signal SIGSEGV, Segmentation fault.
#0  0x00007f38336f866f in OCC::Account::davUser() const () from /opt/ownCloud/ownCloud/lib/x86_64-linux-gnu/libowncloudsync.so.0
#0  0x00007f38336f866f in OCC::Account::davUser() const () from /opt/ownCloud/ownCloud/lib/x86_64-linux-gnu/libowncloudsync.so.0
#1  0x00007f38336f86d9 in OCC::Account::davPath() const () from /opt/ownCloud/ownCloud/lib/x86_64-linux-gnu/libowncloudsync.so.0
#2  0x00005573cef77295 in ?? ()
#3  0x00007f3832b07083 in __libc_start_main (main=0x5573cef76bf0, argc=4, argv=0x7fffbf45bdb8, init=<optimized out>, fini=<optimized out>, rtld_fini=<optimized out>, stack_end=0x7fffbf45bda8) at ../csu/libc-start.c:308
#4  0x00005573cef784be in ?? ()
```
</details>

**Comment by saw-jan** [comment]
> While running tests against client 3.0, command /usr/bin/owncloudcmd --trust crashes. Here are the logs:

Possible fix PR https://github.com/owncloud/client/pull/10150

**Comment by saw-jan** [comment]
Possible fix PR has been merged. Good to go ahead.

**Comment by saw-jan** [comment]
Tests against client `3.0` fails with error:
```bash
2022-10-10 11:35:21,872 - WARNING - sharer - Non-zero exit code 1 from command '/usr/bin/owncloudcmd --trust /smashdir/test_reshareDir/sharer/ owncloud://test_reshareDir1:owncloud@owncloud-server:8080/remote.php/webdav/ >> /smashdir/test_reshareDir/sharer-ocsync.step03.cnt000.log 2>&1'
```
<details>
<summary>Log</summary>

```bash
10-10 11:35:21:864 [ warning sync.checkserverjob ]:	error: status.php replied 404
10-10 11:35:21:864 [ fatal default ]:	Failed to resolve http://owncloud-server:8080/remote.php/webdav Error: Error transferring http://owncloud-server:8080/remote.php/webdav/status.php - server replied: Not Found.
Aborted (core dumped)
```
</details>

<details>
<summary>Coredump</summary>

```bash
Excess command line arguments ignored. (full)
[New LWP 1492]
[New LWP 1495]
[New LWP 1493]
[New LWP 1494]
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".
Core was generated by `/usr/bin/owncloudcmd --trust /smashdir/test_reshareDir/sharer/ owncloud://test_'.
Program terminated with signal SIGABRT, Aborted.
#0  __GI_raise (sig=sig@entry=6) at ../sysdeps/unix/sysv/linux/raise.c:50
50	../sysdeps/unix/sysv/linux/raise.c: No such file or directory.
[Current thread is 1 (Thread 0x7f5bef1a8840 (LWP 1492))]
#0  __GI_raise (sig=sig@entry=6) at ../sysdeps/unix/sysv/linux/raise.c:50
#1  0x00007f5bf2bc3859 in __GI_abort () at abort.c:79
#2  0x00007f5bf300ec39 in QMessageLogger::fatal(char const*, ...) const () from /opt/ownCloud/qt-5.15.2/lib/x86_64-linux-gnu/libQt5Core.so.5
#3  0x0000557c1be006ac in ?? ()
#4  0x00007f5bf3274a4e in ?? () from /opt/ownCloud/qt-5.15.2/lib/x86_64-linux-gnu/libQt5Core.so.5
#5  0x00007f5bf3875ad3 in ?? () from /opt/ownCloud/ownCloud/lib/x86_64-linux-gnu/libowncloudsync.so.0
#6  0x00007f5bf3274a4e in ?? () from /opt/ownCloud/qt-5.15.2/lib/x86_64-linux-gnu/libQt5Core.so.5
#7  0x00007f5bf35f0cc6 in ?? () from /opt/ownCloud/qt-5.15.2/lib/x86_64-linux-gnu/libQt5Network.so.5
#8  0x00007f5bf369896e in ?? () from /opt/ownCloud/qt-5.15.2/lib/x86_64-linux-gnu/libQt5Network.so.5
#9  0x00007f5bf326b5c9 in QObject::event(QEvent*) () from /opt/ownCloud/qt-5.15.2/lib/x86_64-linux-gnu/libQt5Core.so.5
#10 0x00007f5bf323d9ff in QCoreApplication::notifyInternal2(QObject*, QEvent*) () from /opt/ownCloud/qt-5.15.2/lib/x86_64-linux-gnu/libQt5Core.so.5
#11 0x00007f5bf3240331 in QCoreApplicationPrivate::sendPostedEvents(QObject*, int, QThreadData*) () from /opt/ownCloud/qt-5.15.2/lib/x86_64-linux-gnu/libQt5Core.so.5
#12 0x00007f5bf3298cd7 in ?? () from /opt/ownCloud/qt-5.15.2/lib/x86_64-linux-gnu/libQt5Core.so.5
#13 0x00007f5bf140017d in g_main_context_dispatch () from /lib/x86_64-linux-gnu/libglib-2.0.so.0
#14 0x00007f5bf1400400 in ?? () from /lib/x86_64-linux-gnu/libglib-2.0.so.0
#15 0x00007f5bf14004a3 in g_main_context_iteration () from /lib/x86_64-linux-gnu/libglib-2.0.so.0
#16 0x00007f5bf3298332 in QEventDispatcherGlib::processEvents(QFlags<QEventLoop::ProcessEventsFlag>) () from /opt/ownCloud/qt-5.15.2/lib/x86_64-linux-gnu/libQt5Core.so.5
#17 0x00007f5bf323c53b in QEventLoop::exec(QFlags<QEventLoop::ProcessEventsFlag>) () from /opt/ownCloud/qt-5.15.2/lib/x86_64-linux-gnu/libQt5Core.so.5
#18 0x00007f5bf32446e4 in QCoreApplication::exec() () from /opt/ownCloud/qt-5.15.2/lib/x86_64-linux-gnu/libQt5Core.so.5
#19 0x0000557c1bdfdba9 in ?? ()
#20 0x00007f5bf2bc5083 in __libc_start_main (main=0x557c1bdfcd60, argc=4, argv=0x7fff0eb4b8c8, init=<optimized out>, fini=<optimized out>, rtld_fini=<optimized out>, stack_end=0x7fff0eb4b8b8) at ../csu/libc-start.c:308
#21 0x0000557c1bdfe53e in ?? ()
```
</details>

**Comment by saw-jan** [comment]
~Blocked until https://github.com/owncloud/docs-client-desktop/issues/279#issuecomment-1274592308~

File: .drone.star [hunk 3]
```diff
@@ -1,13 +1,28 @@
+PLUGINS_SLACK = "plugins/slack:1"
+PLUGINS_DOWNLOAD = "plugins/download"
+LIBRARY_MARIADB = "library/mariadb:10.3"
+LIBRARY_REDIS = "library/redis:4.0"
+OC_UBUNTU = "owncloud/ubuntu:20.04"
+OC_BASE = "owncloud/base:20.04"
+OC_CI_DRONE_CANCEL_PREVIOUS_BUILDS = "owncloudci/drone-cancel-previous-builds"
+OC_DOCKER_SMASHBOX = "sawjan/smashbox:appimage"
```
Comment (saw-jan) [inline .drone.star]: **[NOTE]** change to owncloud image
```suggestion
OC_DOCKER_SMASHBOX = "owncloud/smashbox:latest"
```

**Comment by saw-jan** [comment]
Depends on these PRs: https://github.com/owncloud-docker/smashbox/pull/58 and https://github.com/owncloud/smashbox/pull/174

**Comment by saw-jan** [comment]
Also, need this [issue](https://github.com/owncloud/client/issues/10185) to be fixed as the smashbox docker image is currently using `apt` to install desktop client

**Comment by SagarGi** [comment]
@saw-jan putting this to backog for now.

**Comment by saw-jan** [comment]
Will re-open if needed

**Pull request closed without merging**